Jellyfin.Api/Controllers/LiveTvController.cs

```diff
@@ -1015,7 +1014,7 @@ public ActionResult<ListingsProviderInfo> GetDefaultListingProvider()
             if (!string.IsNullOrEmpty(pw))
             {
                 using var sha = SHA1.Create();
-                listingsProviderInfo.Password = Hex.Encode(sha.ComputeHash(Encoding.UTF8.GetBytes(pw)));
+                listingsProviderInfo.Password = Convert.ToHexString(sha.ComputeHash(Encoding.UTF8.GetBytes(pw)));
```

Comment:
```suggestion
                listingsProviderInfo.Password = Convert.ToHexString(sha.ComputeHash(Encoding.UTF8.GetBytes(pw))).ToLower();
```

SchedulesDirect requires lowercase hex